services/autoscaling/src/simcore_service_autoscaling/utils_aws.py

```diff
@@ -0,0 +1,103 @@
+""" Free helper functions for AWS API
+
+"""
+
+import logging
+import time
+from textwrap import dedent
+from typing import Final
+
+import boto3
+
+from .core.settings import AwsSettings
+
+logger = logging.getLogger(__name__)
+
+
+AWS_EC2: Final = [
```

Comment:
Possible future improvement: Get this list programmatically instead of hardcoded

Reply:
yep. I guess it is feasible. Will add a note

Reply:
That was my first though but I think we don't want that - except if we have an algorithm which also calculate the best price/perf adapted to our need (which I did manually to create this list). 

The machine listed there are the one who correspond the most to our needs while being the cheapest. Downside is that it need to be checked from time to time.